BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoNotAppliedView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  SkontoNotAppliedView.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+class SkontoNotAppliedView: UIView {
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        let title = NSLocalizedStringPreferredGiniBankFormat("ginibank.skonto.notapplied.title",
+                                                             comment: "Without Skonto discount")
+        label.text = title
+        label.accessibilityValue = title
+        label.textColor = .giniColorScheme().text.primary.uiColor()
+        label.font = configuration.textStyleFonts[.bodyBold]
+        label.adjustsFontForContentSizeCategory = true
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var statusLabel: UILabel = {
+        let label = UILabel()
+        let title = NSLocalizedStringPreferredGiniBankFormat("ginibank.skonto.info.status",
+                                                             comment: "• Active")
+        let font = configuration.textStyleFonts[.footnoteBold] ?? .systemFont(ofSize: 17)
```

Comment:
[clarification] Same question here. this `systemFont(ofSize: 17)` is not our ` .footnoteBold: UIFontMetrics(forTextStyle: .footnote).scaledFont(for: UIFont.boldSystemFont(ofSize: 13))`